src/quo2/components/list_items/dapp/component_spec.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo2.components.list-items.dapp.component-spec
+  (:require [quo2.components.list-items.dapp.view :as dapp]
+            [test-helpers.component :as h]))
+
+(h/describe
+  "dapp test"
+  (h/test "renders default dapp"
+    (h/render [dapp/view
+               {:dapp                {:avatar nil
+                                      :name   "Coingecko"
+                                      :value  "coingecko.com"}
+                :state               :default
+                :action              :icon
+                :blur?               false
+                :customization-color :blue
+                :on-press            (h/mock-fn)}])
+    (h/is-truthy (h/get-by-text "coingecko.com"))))
```

Comment:
It would be nice if you can add more tests, like on-press event and name.

Reply:
Thanks for you review @ajayesivan!

done.